worker/include/PayloadChannel/PayloadChannelSocket.hpp

```diff
@@ -32,6 +33,7 @@ namespace PayloadChannel
 		Listener* listener{ nullptr };
 		// Others.
 		size_t msgStart{ 0u }; // Where the latest message starts.
+		uint8_t* readBuffer;
```

Comment:
Let's follow code convention:

```suggestion
		// Allocated by this.
		uint8_t* readBuffer{ nullptr };
		// Others.
		size_t msgStart{ 0u }; // Where the latest message starts.
```

Reply:
I'm gonna apply this and merge.